src/cares_wrap.cc

```diff
@@ -69,6 +69,19 @@ using v8::Value;
 
 namespace {
 
+inline uint16_t cares_get_16bit(const unsigned char* p) {
```

Comment:
Hmmmm, this was referred to https://github.com/nodejs/node/blob/master/deps/cares/src/ares_dns.h#L23-L35 and https://github.com/nodejs/node/blob/master/deps/cares/src/ares_parse_soa_reply.c#L109-L113. I just try to keep them similar to show that this code is referred from there.